components/rmrk/Gallery/Flipper.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <CommonHolderTable
+    :table-rows-option="flipperTableRowList"
+    group-key-option="Flipper"
+    openOnDefault
+    date-header-label="Last Activity"
```

Comment:
`CODE_STYLE.md` is sad in the corner :/

Reply:
: )  Sorry, do you mean the name of the attribute is not clear enough? Or need to add a translation key?

Reply:
`dateHeaderLabel="Last Activity"`

Reply:
BTW, actually, kebab-case is recommended by vue.  I think we should follow this rule. 👀 

<img width="816" alt="image" src="https://user-images.githubusercontent.com/31397967/162614508-cd0f160b-0263-47b3-87a7-1d5db17f6690.png">
https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing